assets/js/script.js

```diff
@@ -87,17 +135,57 @@ const recipeCardGenerator = (recipe1, recipe2, recipe3) => {
 		recipeHeaderDiv.setAttribute("class", "collapsible-header");
 		// create body div and set class to collapsible-body
 		const recipeBodyDiv = document.createElement("div");
-		recipeBodyDiv.setAttribute("class", "collapsible-body");
+		recipeBodyDiv.classList= "collapsible-body";
 
 		// set content of header and body divs
-		recipeHeaderDiv.innerHTML = `<i class="material-icons">filter_drama</i> ${i}`;
-		recipeBodyDiv.innerHTML = `<span>Lorem ipsum dolor sit amet.</span> ${i}`;
+		recipeHeaderDiv.innerHTML = `<i class="material-icons">filter_drama</i> Recipe ${i}`;
+
+		// create and append row
+		const recipeBodyRow = document.createElement('div');
+		recipeBodyDiv.append(recipeBodyRow);
+
+		// create and append text div
+		const recipeTextDiv = document.createElement('div');
+		recipeTextDiv.setAttribute("class", "col s11 m11 l11 xl11");
+		recipeBodyRow.append(recipeTextDiv);
+
+		// create recipe span
+		const recipeSpan = document.createElement('span');
+		recipeSpan.textContent = 'Lorem ipsum dolor sit amet.';
+		recipeTextDiv.append(recipeSpan);
+
+		// create like button div
+		const likeButtonDiv = document.createElement('div');
+		likeButtonDiv.setAttribute('class', 'col s1 m1 l1 xl1 favorites');
+		recipeBodyRow.append(likeButtonDiv);
+
+		// create like button
+		const likeButton = document.createElement('i');
+		likeButton.setAttribute('class', 'material-icons');
+		likeButton.textContent = 'favorite_border';
+		likeButtonDiv.append(likeButton);
+		likeButton.addEventListener("click", function() {
+			likeButton.textContent = 'favorite';
+		})
 
 		// append everything
 		recipeLi.append(recipeHeaderDiv, recipeBodyDiv);
 		recipeUl.append(recipeLi);
 	}
+	// append recipeCard container div to contentrow in main
 	$("#contentRow").append(recipeCardDiv);
+
+	// add event listeners
+
+
+};
+
+
+const likeButtonHandler = () => {
+	console.log('hi :)');
```

Comment:
hehe did we actually want to add the functionality below that is commented out?

Reply:
i'm cool with it if we want until a later time to do it, but it looks like the code below _should_ hypothetically work